packages/next-preact/preact-compat.js

```diff
@@ -0,0 +1,39 @@
+const {
+  version,
+  DOM,
+  PropTypes,
+  Children,
+  render,
+  createClass,
+  createFactory,
+  createElement,
+  cloneElement,
+  isValidElement,
+  findDOMNode,
+  unmountComponentAtNode,
+  Component,
+  PureComponent,
+  unstable_renderSubtreeIntoContainer,
+  __spread,
+} = require('preact-compat');
+const { createContext } = require('preact-context');
+
+module.exports = {
```

Comment:
Could this actually be

```
const preact = require('preact-compat')
const {createContext} = require('preact-context')

preact.createContext = createContext
```

Reply:
@timneutkens 
with that proposal, it worked normally.